bt/core.py

```diff
@@ -351,20 +357,17 @@ class StrategyBase(Node):
             this data to determine strategy logic.
 
     """
-
-    _capital = cy.declare(cy.double)
+    
     _net_flows = cy.declare(cy.double)
     _last_value = cy.declare(cy.double)
     _last_notl_value = cy.declare(cy.double)
     _last_price = cy.declare(cy.double)
     _last_fee = cy.declare(cy.double)
-    _paper_trade = cy.declare(cy.bint)
-    _bidoffer_set = cy.declare(cy.bint)
-    bankrupt = cy.declare(cy.bint)    
+    _paper_trade = cy.declare(cy.bint)    
+    bankrupt = cy.declare(cy.bint)
```

Comment:
convention is underscore

Reply:
It is, but the bankrupt attribute has always been like that. I thought about underscoring it and exposing it via a property, but I thought maybe users were setting it externally, i.e. to force bankruptcy (or reverse it) under certain situations.